### PR TITLE
diagnostics: pull in checks docs manually for initial release

### PR DESCRIPTION
**Pull request by xginn8**
until the device-diagnostics repo is open-sourced, these docs need to be pulled in manually

Connects-to: https://github.com/balena-io/device-diagnostics/pull/113
Connects-to: https://github.com/balena-io/device-diagnostics/issues/110
Signed-off-by: Matthew McGinn <matthew@balena.io>
Change-type: minor

### CODE REVIEW

**Comment by xginn8** [comment]
Screenshot_2019-09-11 Check Descriptions - Balena Documentation

**Comment by xginn8** [comment]
maybe this location is the wrong place for these docs, i'm open to suggestions!

File: pages/reference/diagnostics.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Check Descriptions
+
+As part of the diagnostics suite, you will find various checks that can be run on-device. Below is a description of each
```
Comment (shaunmulligan) [inline pages/reference/diagnostics.md]: @xginn8 these check all get run together right? or are we making it so a single command can be run?

File: pages/reference/diagnostics.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+#### Triage
+Upgrade your device to `balenaOS 2.x` (contact support).
+
+### check_under_voltage
+#### Summary
+Often seen on Raspberry Pi devices, these kernel messages indicate that the power supply is insufficient for the device and any peripherals that might be attached. These errors also precede seemingly erratic behavior.
+
+#### Triage
+Replace the power supply with a known-good supply (typically supplying at least 2.5V).
```
Comment (shaunmulligan) [inline pages/reference/diagnostics.md]: 2.5V is wrong, it should rather be 5V at >2.5A

File: pages/reference/diagnostics.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+### check_memory
+#### Summary
+This check simply confirms that a given device is running at a given memory threshold (set to 90% at the moment).
+Oversubsribed memory can lead to OOM events.
```
Comment (shaunmulligan) [inline pages/reference/diagnostics.md]: maybe link to what OOM is

File: pages/reference/diagnostics.md (hunk 2 above)
Comment (xginn8) [reply pages/reference/diagnostics.md]: oops :man_facepalming: i swear i googled this too..

File: pages/reference/diagnostics.md (hunk 1 above)
Comment (xginn8) [reply pages/reference/diagnostics.md]: You are correct, that is strangely worded.

**Comment by xginn8** [comment]
@shaunmulligan addressed your comments at the source, here: https://github.com/balena-io/device-diagnostics/pull/114

**Pull request merged**